app/ide-desktop/lib/dashboard/src/authentication/src/components/svg.tsx

```diff
@@ -0,0 +1,53 @@
+/** @file File containing SVG icon definitions. */
+/** TODO [NP]: https://github.com/enso-org/cloud-v2/issues/342
+ * These should all be regular `.svg` files rather than React components, but React doesn't include
+ * the `svg` files when building for Electron. How do we solve this? */
```

Comment:
The question "How do we solve this?" should rather not be part of the comment. It is TODO, so naturally it is an unresolved question.

Reply:
Fixed.